lib/lrmd/lrmd_alerts.c

```diff
@@ -176,6 +179,12 @@ exec_alert_list(lrmd_t *lrmd, GList *alert_list, enum crm_alert_flags kind,
                                               timestamp);
                 free(timestamp);
             }
+
+            crm_time_set_hr_dt(dt, now);
+            snprintf(timestamp_epoch, 32, "%lld", crm_time_get_seconds_since_epoch(dt));
+            copy_params = alert_key2param(copy_params, CRM_alert_timestamp_epoch, timestamp_epoch);
+            snprintf(timestamp_nanos, 12, "%06d000", now->useconds);
```

Comment:
Now that I'm seeing it, I'm wondering, do you think using nanoseconds is a good idea, or should we use microseconds? Since crm_time_hr_t only supports microseconds, we're unlikely to ever go deeper than that.

Reply:
Hi Ken,

Thanks for your comment.